apps/settings/build/settings.build.jslike

```diff
@@ -9,40 +9,41 @@
   // which makes the objects inaccessible by reference.
   // If the inquiries to the object are all performed by requirejs, we can
   // remove the path of the object from the following list.
+  // organize in alphabet order
   paths: {
     'vendor/jszip': 'empty:',
```

Comment:
A suggestion here, lets also alphabetize here based on the first character of the parent folder (not just file name). So for example modules/... will go before venoder/.... In fact vendor should go closer to the end.